theme/templates/base.html

```diff
@@ -7,6 +7,9 @@
         {% block meta %}{% endblock %}
         {% block twitter_card %}{% endblock %}
         {% block opengraph %}{% endblock %}
+        <!-- MailChimp Newsletter Form Styles -->
+        <link href="//cdn-images.mailchimp.com/embedcode/horizontal-slim-10_7.css" rel="stylesheet" type="text/css">
```

Comment:
Remove this CSS file and adjust the subscribe form styles so it looks like the rest of the website.

Reply:
To make myself more clear: this + the above definitions seem to be fine when previewing locally but break when running a preview for production. I think the disqus styles may be overriding something since the comment box also has a subscribe button. 

Maybe try using different classes and IDs

Reply:
Production styles are still messed up. See below for the command used to generate the production preview.